### DISCUSSION TITLE
Request for updating libc version

### DISCUSSION THREAD

**Discussion by happys2333** [linuxdeepin/developer-center#3488] [Features Request & Ideas | 特性请求 & 头脑风暴]
I think that we should update our system libc. Here is a problem when I try to run victoria3 through steam.
``` shell
./victoria3: /lib/x86_64-linux-gnu/libstdc++.so.6: version `GLIBCXX_3.4.26' not found (required by ./victoria3)
```
This shows our libc are too old.

**Comment by BLumia** [comment]
Should be updated in v23 (alpha version release will be in the upcoming week).

Since glibc update could break a lots of app so we probably won't update it in v20.
